app/src/main/java/net/bible/android/view/activity/navigation/History.kt

```diff
@@ -22,17 +22,19 @@ import android.content.Intent
 import android.os.Bundle
 import android.util.Log
 import android.view.View
+import android.view.ViewGroup
 import android.widget.ArrayAdapter
 import android.widget.ListAdapter
 import android.widget.ListView
+import android.text.format.DateFormat.format
+import kotlinx.android.synthetic.main.history_list_item.view.*
```

Comment:
Is this used (I don't seeusage)? if not, remove

Reply:
https://github.com/AndBible/and-bible/blob/e6975e0c90094156080c4af8d0e9e85fa4a89c81/app/src/main/java/net/bible/android/view/activity/navigation/History.kt#L75-L77

Yes it's used in these lines.